content/700-optimize/200-getting-started.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Getting started'
+metaTitle: 'Getting started with Prisma Optimize'
+metaDescription: 'Learn how to get up and running with Prisma Optimize.'
+tocDepth: 3
+toc: true
+---
+
+## Prerequisites
+
+To get started with Optimize, you will need the following:
+
+- A [Prisma Data Platform workspace](https://console.prisma.io).
+- A project that uses [Prisma Client](/orm/prisma-client) `5.16.1` or higher (We always recommend using the latest version of Prisma).
+- A PostgreSQL, MySQL/MariaDB, PlanetScale, CockroachDB, or MongoDB database.
+
+## 1. Enable Optimize
+
+Login to your [Prisma Data Platform account](https://console.prisma.io) and then navigate to the [ Optimize dashboard](https://optimize.prisma.io).
+
+<Admonition>
+
+You should use Optimize in your development or staging environment.
```

Comment:
@nikolasburk there should be a performance hit as tracing would be enabled. As a standard practice, it would make sense for users to debug the queries and improve perf during staging or development. I'll remove it as during the show and tell it was said users can use it for production. 
 
```suggestion
```